src/components/Selfie/Capture.js

```diff
@@ -9,7 +9,7 @@ import { ToggleFullScreen } from '../FullScreen'
 import Timeout from '../Timeout'
 import Camera from '../Camera'
 import CameraError from '../CameraError'
-import style from './style.css'
+import style from '../Photo/style.css'
```

Comment:
Why is the style of this component in `Photo/` given that the component itself has been moved to a different directory? They should be together. Also we already have a component called Capture. To avoid confusion I would just call this file ` `index.js`. For reference, follow the pattern used by other components

Reply:
So I tried following the pattern on the video component which was my initial implementation. For 2 reasons I was not able to follow it completely. 

1. The video index component controls the flow (or should be - there is a bug) the flow between intro and capture.
2. Because I was trying to follow the video component in my initial implementation I moved the selfie capture from the Capture folder to its own directory. The Capture directory does not have an `index.js` but rather it contains capture related components. This is inconsistent with the rest of the component structure.
3. Having an `index.js` on the Selfie directory which did control the flow between intro and capture I realised (and flagged) the bug on the video back capture navigation. You can see that in the `src/components/Router/StepComponentMap.js ` file. However because step component map controls the creation and navigation between these two components there is no longer a need for `index.js` under Selfie directory.

So if we really want an `index.js` for Selfie which should it be: Intro or Capture? Otherwise I can move back the selfie capture to the Photo directory where it originally was. In the latter case should I have Selfie Intro as a component i.e. `SelfieIntro/index.js`? or just have it under the Photo directory as `SelfieIntro.js`?